src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+  const lines = sourceString.split(';');
+
+  lines.forEach((line) => {
+    const [property, value] = line.split(':');
```

Comment:
When splitting the line by the colon, there might be cases where the colon is missing, which would cause an error. It's important to check that the result of the split operation has at least two elements before destructuring it into `property` and `value`.